docs/source/commandline/commandline-all.rst

```diff
@@ -928,95 +1148,176 @@ Core Video Options
+    Tells MAME how many output windows to create.  For most systems, a single
+    output window is all you need, but some systems originally used multipl
+    screens (*e.g. Darius and PlayChoice-10 arcade machines*).  Each screen
+    (up to 4) has its own independent settings for physical monitor, aspect
+    ratio, resolution, and view, which can be set using the options below.
+
+    The default is ``1``.
 
 .. _mame-commandline-window:
 
 **-[no]window** / **-[no]w**
 
-	Run MAME in either a window or full screen. The default is OFF (**-nowindow**).
+    Run MAME in either a window or full screen.
+
+    The default is OFF (**-nowindow**).
 
 .. _mame-commandline-maximize:
 
 **-[no]maximize** / **-[no]max**
 
-	Controls initial window size in windowed mode. If it is set on, the window will initially be set to the maximum supported size when you start MAME. If it is turned off, the window will start out at the smallest supported size. This option only has an effect when the -window option is used. The default is ON (**-maximize**).
+    Controls initial window size in windowed mode.  If it is set on, the window
+    will initially be set to the maximum supported size when you start MAME.  If
+    it is turned off, the window will start out at the smallest supported size.
+    This option only has an effect when the -window option is used.
+
+    The default is ON (**-maximize**).
 
 .. _mame-commandline-keepaspect:
 
 **-[no]keepaspect** / **-[no]ka**
 
-	Enables aspect ratio enforcement. When this option is on, the system's proper aspect ratio (generally 4:3 or 3:4) is enforced, so you get the system looking like it should. When running in a window with this option on, you can only resize the window to the proper aspect ratio, unless you are holding down the CONTROL key. By turning the option off, the aspect ratio is allowed to float. In full screen mode, this means that all systems will stretch to the full screen size (even vertical systems). In window mode, it means that you can freely resize the window without any constraints. The default is ON (**-keepaspect**).
+    Enables aspect ratio enforcement. When this option is on, the system's
+    proper aspect ratio (generally 4:3 or 3:4) is enforced, so you get the
+    system looking like it should.
+
+    When running in a window with this option on, you can only resize the window
+    to the proper aspect ratio, unless you are holding down the CONTROL key.  By
+    turning the option off, the aspect ratio is allowed to float.  In full
+    screen mode, this means that all systems will stretch to the full screen
+    size (even vertical systems).
 
-	The MAME team heavily suggests you leave this at default. Stretching systems beyond their original aspect ratio will mangle the appearance of the system in ways that no filtering or HLSL can repair.
+    In window mode, it means that you can freely resize the window without any
```

Comment:
I need to test again, but I'm not sure how applicable this is to non-Windows - I think you can resize freely but it gives black borders.

Reply:
I'm going to leave this alone for the moment. Let me know what you find and we'll go from there.